app/assets/stylesheets/users.css.scss

```diff
@@ -33,3 +33,9 @@
     margin-top:30px;
   }
 }
+
+.user-questions-qset-links {
+  &:hover {
+    text-decoration: underline;
+  }
```

Comment:
Where is this being used? I was unable to find it in app/views/users/show.html.erb, which I assume is where it would be utilized. Do you have plans to apply this styling later?